### ISSUE TITLE
RESULTS: nav-filter dynamic location box

### ISSUE THREAD

**Issue by loogart** [loogart/lapoutineweek#56]
<img width="1357" alt="Screen Shot 2021-01-26 at 10 31 47 AM" src="https://user-images.githubusercontent.com/7881400/105866172-b54d4900-5fc1-11eb-8bfd-754753ff77e6.png">

1. Go to the wrapper div around the location and filter buttons with the styles `col-12 col-md-5 col-lg-4 py-2`
2. Replace the class `.row` with the classes `.d-flex.justify-content-between`
3. Replace BOTH button parents with the class '.col-6' to '.flex-fill'
4. On the filter button parent, add the class `ml-3` after the newly added class `flex-fill`
5. One more fix, for the height of the location button, change the Class `custom-select-lg` to `custom-select-md`
6. Call me if any questions lol

Basically, should look like this:

```
<div class="d-flex justify-content-between">
    <div class="flex-fill">
        <div class="btn btn-block btn-outline-secondary text-dark custom-select-md">
             <!-- location button goes here -->
        </div>
    </div>
    <div class="flex-fill ml-3">
         <!-- filter button goes here -->
    </div>
</div>
```
